### DISCUSSION TITLE
deleting an account to which you have lost access.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#129881] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Good afternoon/evening/morning/night. I lost access to my github account and I can't recover it from a 2fa autintefication, I have 2 2fa but it doesn't work. How can I get my past account deleted or how can I contact real github support to get it back?
Or so it can be removed

**Comment by phanduykhai05** [comment]
hi

**Comment by ebndev** [comment, marked as answer]
Hi @FarParticlar,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
